frontend/src/app/task-loader.service.ts

```diff
@@ -1,23 +1,48 @@
 import { Injectable } from '@angular/core';
+import { UUID } from 'angular2-uuid';
 
-export interface TaskInterface {
+export interface Task {
```

Comment:
Better practice in interfaces naming will be with letter I before name, like ITask, IGroup, IUser etc.